frontend/pages/members/[id]/index.tsx

```diff
@@ -38,6 +38,9 @@ export default function MemberPage() {
       id: member?.id,
     },
   });
+  const { refetch: refetchNotifications } = useNotificationsQuery({
+    skip: true, // we don't want to actually fetch, just refetch
+  });
```

Comment:
Is this needed here because another component depends on the data this query returns?

Or in other words, why is this needed here?

Reply:
Yes. If someone pinged you and you received a notification, then ping them back, that notification is deleted. So it has to refetch notifications to update accordingly.